### DISCUSSION TITLE
Avatar Group (Extended)

### DISCUSSION THREAD

**Discussion by BrunoBiagioniNeto** [Telefonica/mistica-design#1104] [Proposals]
- Necesitamos agrupar el componente Avatar de 1 a 5 elementos para que el Borde se active como obligatorio.
- Variantes de cantidad de artículos de 1 a 5. Cuando el número es 5, el último artículo debe ser un Avatar Initials.
- Los tamaños y variantes continúan siendo controlados por el componente Avatar.

![Avatar - Group - New Comp](https://user-images.githubusercontent.com/123083886/223187367-6a64f8c1-5311-40de-9d60-f9c9b4f6fe98.jpg)

**Comment by yceballost** [comment]
Buenas Bruno

Aquí salen varias cosas:
1. Consideramos desde Mística que de momento, esto **debería ser un componente extended** porque no ha salido esta necesidad de agrupar avatares en otro producto, según el proceso definido para incluir componentes en Mística si en el futuro otro equipo necesita esto, aquí es cuando ese componente extended se haría "core" en Mística
2. Para que podáis hacer este componente extended debemos evolucionar otros componentes, en el caso de web el componente `inline` debería aceptar valores negativos (a día de hoy no lo acepta), haremos ticket para desarrollo de esto. #1105 
3. Tendriamos que añadir en Mística la posibilidad de que el avatar lleve border, crearemos ticket tb para esto. #1108
